extensions/pkg/controller/worker/state_actuator.go

```diff
@@ -192,9 +192,7 @@ func addMachineToMachineDeploymentState(machine *machinev1alpha1.Machine, machin
 		Labels:      machine.Labels,
 	}
 	machine.OwnerReferences = nil
-	machine.Status = machinev1alpha1.MachineStatus{
-		Node: machine.Status.Node,
-	}
+	machine.Status = machinev1alpha1.MachineStatus{}
```

Comment:
Is it necessary to explicitly set this here?

Reply:
Okay, this will be reconciled by MCM after migration?

Reply:
We set this explicitly here, because adding the `Status` to the `Worker` stat is not necessary. Without this `Machine.Status` would most likely not be empty.